src/treeview.rs

```diff
@@ -44,21 +44,8 @@ fn main() {
     left_tree.set_headers_visible(false);
     append_text_column(&left_tree);
 
-    // print out when a row is selected
-
-    let left_selection = left_tree.get_selection().unwrap();
-    left_selection.connect_changed(|tree_selection| {
-        let (left_model, iter) = tree_selection.get_selected().unwrap();
-        println!("selected row {}", left_model.get_path(&iter));
-    });
-
     for _ in 0..10 {
-        let iter = left_store.append();
-        left_store.set_value(&iter, 0, &"I'm in a list".to_value());
-
-        // select this row as a test
-        //
-        left_selection.select_path(&left_store.get_path(&iter));
+        left_store.insert_with_values(None, &[0], &[&"I'm in a list"]);
```

Comment:
I think that a comment which explains what happens here would be very appreciated.